internal/local/command_kafka_start.go

```diff
@@ -115,91 +133,121 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	output.Println("\r")
-
 	log.CliLogger.Tracef("Successfully pulled Confluent Local image")
 
-	if err := c.prepareAndSaveLocalPorts(cmd, c.Config.IsTest); err != nil {
+	numBrokers, err := cmd.Flags().GetInt32("brokers")
+	if err != nil {
 		return err
 	}
+	if numBrokers < 1 || numBrokers > 4 {
+		return errors.NewErrorWithSuggestions("Input number of brokers out of range.", "Choose the number of brokers between 1 and 4.") // wording?
+	}
```

Comment:
@brianstrauch  I added a range check for between 1 and 4, but not sure about the error wording here. also is there other examples for enforcing the range of a int flag in our code?

Reply:
How about the following?

> `--num-brokers` must be an integer between 1 and 4, inclusive.

Reply:
+1 to the flag wording.

As for the error, how about just reflecting the flag description, like: `number of brokers must be between 1 and 4, inclusive` and leave off the suggestion.

Reply:
👍